sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -16,13 +16,18 @@ import {
 export const LATEST_API_VERSION = "7.3";
 
 /**
- * The optional parameters accepted by the KeyVault's KeyClient
+ * The optional parameters accepted by the KeyVault's CertificateClient
  */
 export interface CertificateClientOptions extends ExtendedCommonClientOptions {
   /**
    * The accepted versions of the KeyVault's service API.
    */
   serviceVersion?: "7.0" | "7.1" | "7.2" | "7.3";
+
+  /**
+   * Whether to verify the authentication challenge resource matches the Key Vault or Managed HSM domain. The default is true.
```

Comment:
Don't mention "Managed HSM" for certificates or secrets. Only applies to administration and keys packages.